guides/doc-Managing_Security_Compliance/master.adoc

```diff
@@ -6,6 +6,8 @@ include::common/header.adoc[]
 = {ManagingSecurityDocTitle}
 
 ifdef::foreman-deb[]
+== Unavailable for Debian/Ubuntu
+
 This feature is not available for Debian/Ubuntu.
```

Comment:
```suggestion
This feature is not available for hosts running Debian/Ubuntu.
```

Out of scope.

Reply:
https://github.com/theforeman/foreman-documentation/pull/3660